integrated_channels/cornerstone/transmitters/content_metadata.py

```diff
@@ -16,6 +16,18 @@ def __init__(self, enterprise_configuration, client=CornerstoneAPIClient):
             client=client
         )
 
+    def transmit(self, create_payload, update_payload, delete_payload):
```

Comment:
cornerstone transmitter.transmit -> does nothing
cornerstone transmitter.trasnmit_for_web -> does what the normal transmit does